lib/services/product_brutto_price.rb

```diff
@@ -0,0 +1,8 @@
+module Store
+  class ProductBruttoPrice
+    def call(basket_id, product_id)
+      product = FetchProductFromBasket.new.call(basket_id, product_id)
+      product.quantity * product.price * (1 + product.vat)
```

Comment:
The only mathematic symbol that should not be combined with spacebars is `**`. You should always use spacebars around `*`, `+`, ...